### ISSUE TITLE
Determine offerings for badges re-launch

### ISSUE THREAD

**Issue by hannahkane** [mozilla/learning-networks#236]
## DARCI

Driver: @chadsansing 
Accountable: @chrislarry33 
Responsible: @chadsansing 
Consulted: @amirad, @anmechung, @soapdog 
Informed: @hannahkane

Have had this conversation several times, and even came to an agreement before, but it seems like we need to revisit it in light of conversation here https://github.com/mozilla/learning-networks/issues/212#issuecomment-133518603 and here https://github.com/mozilla/teach.webmaker.org/issues/1191#issuecomment-137153170

Badges that have been discussed:
### 21st Century Skills

Lead: @anmechung, Iris
Description: badges corresponding to the four 21st Century Skills that have been identified
Status: getting feedback from Advisory Board; these will definitely be included with initial badge offerings
### Club Captains and Regional Coordinators

Lead: @amirad? 
Consulted: @chadsansing
Description: "Macro" badges that can be leveled up (e.g. bronze -> silver -> gold -> red)
Status: details of leveling up need to be finalized; these will _likely_ be included with initial badge offerings
### Leadership Roles

Lead: Hive leads? MozFest team?
Description: badges for leadership roles in Hive, MozFest (others?)
Status: final confirmation of badges needed; these will _likely_ be included with initial badge offerings 
### Web Literacy

Lead: @chadsansing? @anmechung?
Description: badges corresponding to Web Lit competencies
Status: dependent upon next Web Lit Map iteration

**Comment by amirad** [comment]
cc @soapdog here's where you can plug in on the badges conversation and share your work with badges right now!

**Comment by chadsansing** [comment]
@hannahkane, I am all-aboard here. 21st C and CC/RC seem most actionable with @anmechung & @amirad as consults? Both _can_ be educator facing, & the clubs badges, in particular, would add to MLN leadership work/Mozfest club offerings/NWP training.  

I will +1 your suggestion from [#1191](https://github.com/mozilla/teach.webmaker.org/issues/1191#issuecomment-137038916)

> @kristinashu - for your purposes, let's imagine we have eight badges under a "21st Century Skills" heading and another eight under a "Read, Write, Participate" header. Not because that's what we'll definitely end up with, but rather so you have something specific to work with.

**Comment by hannahkane** [comment]
Added a DARCI to the description. Does that make sense to everyone?

**Comment by chadsansing** [comment]
@chrislarry33, I can drive here in the immediate future for determining badge offerings if that suits. I think we can make this determination quickly. Does my role here work for you? Is there a better driver?

In the meantime, by end of day tomorrow I will ship a 8 21st C & 8 Read/Write/Participate badge scope for comment by end of day next Tuesday. Going for specific starting points --> design exercises as per Hannah's earlier suggestion.

cc @hannahkane

**Comment by chadsansing** [comment]
Can we [start here](https://docs.google.com/a/mozilla.com/spreadsheets/d/1NYOAvlT1G6itAY7p5Q4NUqddQBasvqa0rpx8BMKt1eE/edit?usp=sharing)? Feedback welcome.

cc @hannahkane, @chrislarry33, @anmechung, @amirad, @soapdog, @jgmac1106

**Comment by amirad** [comment]
Sounds good to me. Thanks for leading @chadsansing 
The criteria in that doc looks very specific - was that the intent?

**Comment by anmechung** [comment]
Hannah - your DARCI makes sense to me.  I'm good with leading 21C Skills and co-leading web literacy with Chad.  

Chad: Iris and I have already done some scoping of content for 21C Skills badges. Perhaps you are scoping related?

Criteria in doc - similar question as @amirad.   Would be interested in hearing/understanding more, particularly as this is going to have an impact on the IMLS deliverables.

**Comment by chadsansing** [comment]
@anmechung, please totally replace my best guesses with the work you and Iris have done.

@anmechung & @amirad, I tried to proceed in alignment with Hannah's suggestion that we unblock @kristinashu & start with something specific to begin design with the understanding that is is proof-of-concept work. I wanted to suggest badges that had specific, documentable, if not measurable, outcomes for beginners. I'm sure I'm missing context and history, so plug in any badges you think should already be there that I would have missed.

Maybe we can green-light the clubs ideas and give An-Me and Iris time to switch out any  21st C badges by EOD next Wednesday? And then we'll have our pilot set determined. Feedback?

**Comment by jgmac1106** [comment]
It might make sense that the 8 badges under RWP banner align with 8
activities launched in either  the Web Lit Basics I or Web Lit Basic II:
Privacy teaching kits.

**Comment by chrislarry33** [comment]
Could we get 30 minutes together next week and get on the same page. I think we should do only the minimum we neeed to get the software and design up and running and then test the tool, process and user flow before we get to many scoped. The 21st century and CC/RC badges to me are those prototypes.

**Comment by hannahkane** [comment]
Just wanted to leave notes about what we agreed upon when we met: we're going to lead with 21st Century skills badges and get a basic framework in place for both badge design as well as the basic workflows (which are here: https://redpen.io/ac7c6f096536121da4)

Closing this for now, as I think it's no longer a blocker.
